.travis.yml

```diff
@@ -46,11 +46,17 @@ install:
     fi
   - pwd
   - ls
+  # check if there is a partial download of pyglow, remove if true
+  - if [[ -d "$pyglow_dir" && ! -e "$pyglow_dir/setup.py" && "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      rm -r ./pyglow;
+    fi
+  # pyglow exists, install it  
   - if [[ -d "$pyglow_dir" && -e "$pyglow_dir/setup.py" && "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
       cd pyglow;
       python setup.py install;
       cd ..;
     fi
+  # no pyglow, clone then run script

```

Comment:
Does this go back up to line 49 to check that the clone was successful?

Reply:
If the clone isn't successful then Travis produces an error. I could add code to check for success but at the moment we don't have enough time to clone pyglow and run the pysat tests. The first round, the pyglow clone is made, downloads are performed, and then pyglow is installed. pysat testing continues but the runs out of time. The second time through, with a complete pyglow directory the script just runs the install. This leaves enough time for the pysat testing to complete.

I made a commit since this comment checking the directory size. If it is too small, I delete it all then try to clone.